platform/ios/scripts/release-notes-github.md.ejs

```diff
@@ -6,4 +6,4 @@
 To install this pre-release via a dependency manager, see our [CocoaPods](https://github.com/mapbox/mapbox-gl-native-ios/blob/ios-v<%-CURRENTVERSION%>/platform/ios/INSTALL.md#CocoaPods) or [Carthage](https://github.com/mapbox/mapbox-gl-native-ios/blob/ios-v<%-CURRENTVERSION%>/platform/ios/INSTALL.md#Carthage) instructions.
 <% } -%>
 
-Documentation is [available online](https://www.mapbox.com/ios-sdk/api/<%-CURRENTVERSION%>/) or as part of the download.
+Documentation is [available online](https://docs.mapbox.com/ios/maps/api/<%-CURRENTVERSION%>/) or as part of the download.
```

Comment:
This url is docs.mapbox as opposed to www.Mapbox, just making sure that is correct

Granted I went to the url and it looks right.